lib/rdf.js

```diff
@@ -64,6 +75,15 @@ function createRDF(manifest) {
     delete jetpackMeta["em:icon64URL"];
   }
 
+  if (needsBootstrapJS) {
```

Comment:
Why would you only do this if no bootstrap.js is present? Is there a reason for that if clause?